### DISCUSSION TITLE
Questions about cross-validation

### DISCUSSION THREAD

**Discussion by nebstudio** [MIC-DKFZ/nnUNet#884] [Q&A]
Hello, nnUNet is a very large and comprehensive framework. I would like to ask what is the relationship between the post-processing results after each cross-validation in nnUNet and the results in each epoch?

**Comment by FabianIsensee** [comment]
The postprocessing results after each fold can be ignored. I just used that during development where I didn't want to run all the folds

**Comment by nebstudio** [comment]
Okay, thank you for your answer. I also want to ask you about the brats2020 data set. I saw a lot of classes about processing brats in the code. I used the nnUNetTrainerV2BraTSRegions_DA4_BN_BD class and found that the effect is very bad. May I ask the final processing Where is the code for brats?

**Comment by FabianIsensee** [comment]
Can you please be more specific? You can download the pretrained weights as well if you want. Thay should give you 100% the results we reported in the paper

**Comment by FabianIsensee** [comment]
`nnUNet_download_pretrained_model Task082_BraTS2020`

**Reply by nebstudio** [reply]
Ok thanks for your answer
